doc/api/deprecations.md

```diff
@@ -660,6 +660,15 @@ Type: Runtime
 
 `REPLServer.parseREPLKeyword()` was removed from userland visibility.
 
+<a id="DEP00XX"></a>
+### DEP00XX: path._makeLong()
+
+Type: Runtime
```

Comment:
A compromise approach, then, would be docs-only in 9.x, runtime deprecation in 10.x.